defactor-com/documentation
# content(react-components): improve documentation

### Improve documentation by component

### What does this PR do?

- Resolve #93

### Steps to test

1. Run `yarn build` and `yarn start`
2. Check changes in Front End UI / Components

Now:
![imagen](https://github.com/defactor-com/documentation/assets/49013556/a4fb31fa-b59e-429e-ba6f-252e9e906916)
![imagen](https://github.com/defactor-com/documentation/assets/49013556/8c902655-cd86-426d-a421-aff218bce1df)
![imagen](https://github.com/defactor-com/documentation/assets/49013556/d0b8f4a3-6088-4af5-b0f0-7a796310e2b3)


#### CheckList
